### PR TITLE
chore: remove vestigial code

### PR DESCRIPTION
**Pull request by lemmih**
## Summary of changes

Changes introduced in this pull request:
- `frc46_token` is unused, remove it.
- Re-exported signatures are unused, remove them.
- Remove `Inner` trait.

## Reference issue to close (if applicable)

Closes

## Change checklist

- [x] I have performed a self-review of my own code,
- [x] I have made corresponding changes to the documentation,
- [x] I have added tests that prove my fix is effective or that my feature works (if possible),
- [x] I have made sure the [CHANGELOG][1] is up-to-date. All user-facing changes should be reflected in this document.

[1]: https://github.com/ChainSafe/forest/blob/main/CHANGELOG.md

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -59,7 +59,6 @@ fil_actor_verifreg_state = "6"
 fil_actors_shared = "6"
 filecoin-proofs-api = { version = "14.0", default-features = false }
 flume = "0.10"
-frc46_token = "7"
```
Comment (aatifsyed) [inline Cargo.toml:62]: How did you find this? Do we need a better unused deps story?

File: Cargo.toml [hunk 2]
```diff
@@ -71,7 +70,7 @@ fvm_ipld_blockstore = "0.2"
 fvm_ipld_car = "0.7"
 fvm_ipld_encoding = "0.4"
 fvm_ipld_hamt = "0.7"
-fvm_shared2 = { package = "fvm_shared", version = "~2.5", default-features = false, features = ["testing"] }
+fvm_shared2 = { package = "fvm_shared", version = "~2.5", default-features = false, features = [] }
```
Comment (aatifsyed) [inline Cargo.toml:73]: How do you feel about removing the `default-features` and `features` keys? There are no default features, and there's no point having an empty features array

**Review by aatifsyed — APPROVED** [review]

File: src/cli/subcommands/fetch_params_cmd.rs [hunk 3]
```diff
@@ -51,14 +51,14 @@ fn ram_to_int(size: &str) -> anyhow::Result<SectorSize> {
...
-    type SectorSize = <crate::shim::sector::SectorSize as Inner>::FVM;
+    type SectorSize = crate::shim::sector::SectorSize;
 
     match trimmed {
-        "2048" | "2Ki" | "2ki" => Ok(SectorSize::_2KiB.into()),
-        "8388608" | "8Mi" | "8mi" => Ok(SectorSize::_8MiB.into()),
-        "536870912" | "512Mi" | "512mi" => Ok(SectorSize::_512MiB.into()),
-        "34359738368" | "32Gi" | "32gi" => Ok(SectorSize::_32GiB.into()),
-        "68719476736" | "64Gi" | "64gi" => Ok(SectorSize::_64GiB.into()),
+        "2048" | "2Ki" | "2ki" => Ok(SectorSize::_2KiB),
```
Comment (ruseinov) [inline src/cli/subcommands/fetch_params_cmd.rs:57]: Can't help but wonder if we need to support 3 representations of each sector size. At the very least we could ignore the case and have it `to_lower`.

**Review by ruseinov — APPROVED** [review]
looks good to me apart from one minor nit and some deps comments that @aatifsyed has already made. We should definitely remove all the unused flags in Cargo.toml here or in a follow-up. And look into a tool that would check for unused dependencies too?

File: Cargo.toml (hunk 1 above)
Comment (lemmih) [reply Cargo.toml:62]: I want to use `cargo udeps` again.

**Pull request merged**